src/lib/localStorage.ts

```diff
@@ -0,0 +1,13 @@
+import { IResponseData } from '../components/hooks/useProvideAuth';
+
+export const getUserFromLocalStorage = () => {
+  return localStorage.getItem(`${process.env.REACT_APP_LOCALSTORAGE_KEY}`);
```

Comment:
I meant jusa a const in the same file above all the function definitions. This variable won't be dependent on the environment
const USER_DATA_LOCALSTORAGE = "string"